### DISCUSSION TITLE
ArmV9 support?

### DISCUSSION THREAD

**Discussion by bluewave41** [sensepost/objection#541] [General]
I've got a lot of reports about a modified app I have that it doesn't run on Samsung Galaxy S22 ultra devices with one person citing it's possibly because it runs armv9? I don't know the specifics because I don't have an S22 but does this sound at all plausible and is support being considered?

**Comment by leonjza** [comment]
Architecture support is 100% based on what Frida supports. objection just talks to whichever Frida server it can find.
